### DISCUSSION TITLE
2020/04/fix-t650-windows-10/

### DISCUSSION THREAD

**Discussion by SeanKilleen** [SeanKilleen/seankilleen.github.io#642] [Specific Post Comments]
https://seankilleen.com/2020/04/fix-t650-windows-10/

**Comment by SeanKilleen** [comment]
## Historical Comments

2020-08-29, Disqus user [Goldbero](https://disqus.com/by/godbero/):

> Hello Sean,
> 
> I did not find your post until after I installed the first firmware hit you mentioned on Logitech's site. When things didn't work that great on my T650 I started looking for answers and found your post.
> 
> It was just like you said with the later firmware at the bottom. However, my "Update Firmware" button is stilled grayed out. I cannot figure out how to update to the later firmware. If you have any ideas please send them my way.
> 
> Anyone else trying this may want to avoid installing the default Logitech firmware first.
> 
> Thanks,
> -->Rob

2020-09-12, Disqus user [CUZPAIN IDSFN](https://disqus.com/by/cuzpainidsfn/), responding to Goldbero:

> Yes it will be grayed out. you have to download exe firmware from the support page then you open it. nothing will happen in the front, but behind the scenes ... if you go back to the unifying software you'll see that the button is not grayed out anymore.
> 
> also another trick is to run setpoint on windows7 compatibility mode.
> 
> Good luck :)

2020-09-14, Disqus User [Jon_C](https://disqus.com/by/disqus_xSkNWWfZpl/):

> Hi Sean; thanks for that info - managed to update my (barely) used T650 using your excellent instructions - cheers!
> 
> Jon

2020-09-30, Disqus user [Reker Chen](https://disqus.com/by/rekerchen/):

> Updated the firmware, and my T650 died....The light flash between green and red, and Unifying can't find the touchpad

2020-10-22, Disqus user [infiniti_chik](https://disqus.com/by/infiniti_chik/):

> Hi Sean.....Could you please tell me where to find the "Logitech Receiver App"? Thank you.

With a follow up:

> I found the App and was able to follow your instructions to get my T650 working - I'm so happy now - thanks so much for this article!

2021-01-30, Disqus User "Suzanne, Baltimore, MD":

> Sean, Thanks so much for posting the info concerning the t650. I was able to follow your instructions, and successfully paired the t650 with my laptop. Great help.

2021-06-13, Disqus User [Icaro Icaro](https://disqus.com/by/icaro_icaro/):

> Hi Sean, i've tried the firmware update tool, but neither the 2.5.108413 version or the 1.2.169 version worked (i've tried only the 64 bit versions)... everytime, the update stop at 96%, and the window of the update tool became red... than after 9 seconds it tries again the update, and finish it, but the firmware of the t650 remains the 041.000.00033, i can't find a way to get the 041.000.00038... i also tried to uninstall the setpoint and install a previous version... nothing has changed. Any ideas? Thanks!

**Reply by sh0nuff** [reply]
Surprised to see this and to see so many people discussing this little peripheral - I just got a BNIB one on ebay and elated to find this here, thanks, although I do wonder how to make my desktop see it as a touchpad in Windows settings
